jsx/InstrumentForm.js

```diff
@@ -20,9 +20,16 @@ const InstrumentForm = ({instrument, data, context, options, onUpdate, onSave})
             console.log(e.message || e);
             return false;
           }
-        }).map((element, index) => (
-          renderElement(element, index, data, onUpdate)
-        ))
+        }).map((element, index) => {
+          if(element.Type === 'radio' || element.Type === 'text' || element.Type === 'checkbox' || element.Type === 'select') {
```

Comment:
Can we declare all the input types near the top of the file, above the InstrumentForm definition:
```
const INPUT_TYPES = ['radio', 'text', 'checkbox', 'select', 'date'];
```

Then on this line you can do: 
```
if (INPUT_TYPES.includes(element.Type) { ... }
```